gohilReOneTicket/journal/metadata.tex

```diff
@@ -37,7 +37,8 @@
 \def \authorsSHORT{Gohil, Narayanan and Jain}
 \title{\articleTITLE}
 \date{}
-\author[1]{Varun Gohil}
-\author[1,]{S. Deepak Narayanan}
-\author[1,]{Atishay Jain}
-\affil[1]{Indian Institute of Technology, Gandhinagar, Gandhinagar, India}
+\author[1$\dagger$]{Varun Gohil}
```

Comment:
compilation fails here. there should be a comma between 1 and `$\dagger$`

Reply:
Done